app/models/concerns/sectionable.rb

```diff
@@ -17,16 +17,16 @@ def has_sections(new_sections)
     end
   end
 
-  def get_section(title:)
-    self.sections.where(title: title)
-  end
+  # def get_section(title:)
+  #   self.sections.where(title: title)
+  # end
```

Comment:
@jasonoverby If it looks like this isn't being used anywhere, let's just take it out. It's always in the git-history if we need it back.